leakcanary-android/src/main/java/com/squareup/leakcanary/internal/LeakCanaryInternals.java

```diff
@@ -50,6 +54,8 @@
 
   private static final Executor fileIoExecutor = newSingleThreadExecutor("File-IO");
 
+  private static final String notificationChannelId = "leakcanary";
```

Comment:
This was mirroring the case of the `fileIoExecutor` field right above. I changed it now, but out of curiosity, could you comment on the reason behind the stylistic difference for the Executor field?